ibis/omniscidb/tests/conftest.py

```diff
@@ -17,6 +20,12 @@
 
 @pytest.fixture(scope='module')
 def con():
+    """Define a connection fixture.
```

Comment:
we may want to consider loosening the restriction for test code. you shouldn't necessarily need to write a docstring for a test method

Reply:
IIRC the default for `match` parameter exclude tests for the checking. I just add that to help people to understand easily (as much as possible)  the tests. not sure if it is the right direction ... 

also there is an issue related to the documentation and tests experiences: https://github.com/ibis-project/ibis/issues/1989